### PR TITLE
Adding Z2M Devices

### LINKED ISSUE #1232: Zigbee2mqtt: Add device LED1934G3
```
{
  "name": "0x680ae2fffe4c9018",
  "external_id": "zigbee2mqtt:0x680ae2fffe4c9018",
  "model": "LED1934G3",
  "features": [],
  "should_poll": false,
  "service_id": "8f5634d8-e97e-43f8-806f-bd8ed12a4811",
  "supported": false
}
```

### LINKED ISSUE #1255: Zigbee2mqtt: Add device LED1837R5
HI,
```
{
  "name": "0x804b50fffe6b749e",
  "external_id": "zigbee2mqtt:0x804b50fffe6b749e",
  "model": "LED1837R5",
  "features": [],
  "should_poll": false,
  "service_id": "9f41080f-72be-4813-b30b-0cfdd0dc71aa",
  "supported": false
}
```

### LINKED ISSUE #1260: Zigbee2mqtt: Add device TS0503B
```
{
  "name": "Strip Led Controller",
  "external_id": "zigbee2mqtt:Strip Led Controller",
  "model": "TS0503B",
  "features": [],
  "should_poll": false,
  "service_id": "ad2d582c-9e88-472c-9259-3f893eba6f72",
  "supported": false
}
```

### LINKED ISSUE #1268: Zigbee2mqtt: Add device LED1924G9
```
{
  "name": "0xec1bbdfffe70cef0",
  "external_id": "zigbee2mqtt:0xec1bbdfffe70cef0",
  "model": "LED1924G9",
  "features": [],
  "should_poll": false,
  "service_id": "1df7e896-082a-4009-9e3e-e68a04c7c254",
  "supported": false
}
```

### PR DESCRIPTION
**Pull request by VonOx**
* Partial fix for #1269 
* Fix #1268
* Fix #1232
* Fix #1260 
* Fix #1255

### CODE REVIEW

**Review by Pierre-Gilles — DISMISSED** [review]
Thanks for your PR! Looks good to me (I don't have a strong Zigbee2mqtt expertise so if @cicoub13 you have a feedbacks on this don't hesitate :D)

**Comment by cicoub13** [comment]
Approved for all expected the battery feature, it's auto detected and added to devices with battery.

I'm adding an answer in the issue https://github.com/GladysAssistant/Gladys/issues/1254

File: server/services/zigbee2mqtt/model/Xiaomi.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const Xiaomi = {
     WSDCGQ01LM: [features.temperature, features.humidity],
     WSDCGQ11LM: [features.temperature, features.humidity, features.pressure],
     RTCGQ01LM: [features.motion],
-    RTCGQ11LM: [features.motion, features.illuminance, features.temperature, features.voltage],
+    RTCGQ11LM: [features.motion, features.illuminance, features.temperature, features.voltage, features.battery],
```
Comment (cicoub13) [inline server/services/zigbee2mqtt/model/Xiaomi.js]: It's autodetected by the service (here https://github.com/GladysAssistant/Gladys/blob/3cb07f3753b277a3ac2331327889d919e28efe5b/server/services/zigbee2mqtt/utils/convertDevice.js#L14).

**Review by cicoub13 — CHANGES_REQUESTED** [review]

**Review by cicoub13 — APPROVED** [review]

**Comment by Pierre-Gilles** [comment]
Merged, thanks! 🙏

**Pull request merged**